### PR TITLE
refactor(platform): update provider check helmrelease conditionFunc

### PR DESCRIPTION
**Pull request by wl-chen**
…ollImmediate conditionFunc

**What type of PR is this?**
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespace from that line:
>
> /kind feature

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

**Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.**:

```docs

```

### CODE REVIEW

**Review by huxiaoliang — APPROVED** [review]

File: pkg/platform/provider/baremetal/cluster/create.go [hunk 1]
```diff
@@ -1660,8 +1660,8 @@ func (p *Provider) EnsureCheckAnywhereSubscription(ctx context.Context, c *v1.Cl
 	if err != nil {
 		return err
 	}
-	for i, feed := range sub.Spec.Feeds {
-		_ = wait.PollImmediate(5*time.Second, 1*time.Minute, func() (bool, error) {
+	_ = wait.PollImmediate(5*time.Second, 10*time.Minute, func() (bool, error) {
```
Comment (DanielXLee) [inline pkg/platform/provider/baremetal/cluster/create.go:1663]: The inspection frequency is too high， recommend 30s

Comment (wl-chen) [reply pkg/platform/provider/baremetal/cluster/create.go]: 30s is a little bit long, 15s?

Comment (DanielXLee) [reply pkg/platform/provider/baremetal/cluster/create.go]: 15s ok for me

**Review by DanielXLee — APPROVED** [review]
/lgtm

**Pull request merged**